<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseEditorWindow</class>
 <widget class="QDialog" name="DatabaseEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edytor bazy danych</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_titleTop">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_title">
     <item>
      <spacer name="horizontalSpacer_titleLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_title">
       <property name="font">
        <font>
         <family>Quicksand Medium</family>
         <pointsize>22</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: rgb(26, 95, 180);
}</string>
       </property>
       <property name="text">
        <string>Edytor bazy danych</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_titleRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_titleBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget_main">
     <property name="font">
      <font>
       <family>Ubuntu Light</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane{
border: 2px solid;
}
QTabWidget::tab-bar{
left: 450px;
}
QTabBar::tab{
	min-width: 100px;
	min-height: 35px;
	background-color: rgb(185, 185, 185);
color: black;
	border: 2px solid rgb(110, 110, 110);
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
}
QTabBar::tab:hover{
	background-color: rgb(215, 215, 215);
	border-color: rgb(127, 127, 127);
}
QTabBar::tab:selected{
	background-color: rgb(250, 250, 250);
	border-color: rgb(142, 142, 142);
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab_jumpers">
      <attribute name="title">
       <string>Zawodnicy</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea_jumpers">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>530</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QScrollArea{
background: transparent;
}
QScrollBar::handle{
border: 2px solid rgb(101, 23, 23);
background-color: rgb(255, 229, 190);
height: 10px;
width: 5px;
}</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>528</width>
            <height>519</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>470</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_jumpers"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1057</width>
           <height>78</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_hills">
      <attribute name="title">
       <string>Skocznie</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
